docs/docs/adding-tags-and-categories-to-blog-posts.md

```diff
@@ -211,18 +211,18 @@ exports.createPages = ({ actions, graphql }) => {
     // Tag pages:
     let tags = []
     // Iterate through each post, putting all found tags into `tags`
-    _.each(posts, edge => {
-      if (_.get(edge, "node.frontmatter.tags")) {
-        tags = tags.concat(edge.node.frontmatter.tags)
-      }
-    })
+    posts.forEach(edge => {
+			if (edge.node.frontmatter.tags) {
```

Comment:
I'm guessing this line fails in CircleCI build.

Reply:
Resolved the linting errors